src/rosdiscover/recover/symbolic.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import annotations
+from ast import operator
```

Comment:
this should fail the CI? move to appropriate import group below (i.e., stdlib, third-party, package)

Reply:
fixed